odoo/tools/translate.py

```diff
@@ -1470,28 +1470,30 @@ def load_language(cr, lang):
     installer.lang_install()
 
 
+def get_po_paths(module_name: str, lang: str):
+    lang_base = lang.split('_')[0]
+    langs = [lang_base, lang]  # the latter will overwrite the former
+    if lang_base == "es" and lang != "es_ES":
+        # force es_419 as fallback language for the spanish variations
+        langs[0] = "es_419"
+        if lang == "es_419":
+            langs = langs[:-1]
```

Comment:
nitpicking
I think showing all three results for `langs` explictly can improve the readability
```py
lang_base = lang.split('_')[0]
if lang_base == 'es' and lang != 'es_ES':
    # force es_419 as fallback language for the spanish variations
    if lang == 'es_419':
        langs = ['es_419']
    else:
        langs = ['es_419', lang]
else:
    langs = [lang_base, lang]
```